### PR TITLE
   [CARBONDATA-3173]  add hive-guide and other guides to the root of the file ReadMe

### PR DESCRIPTION
**Pull request by BeyondYourself**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [x] Any interfaces changed?
 
 - [x] Any backward compatibility impacted?
 
 - [x] Document update required?

 - [x] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [x] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by CarbonDataQA** [comment]
Can one of the admins verify this patch?

**Comment by BeyondYourself** [comment]
add the hive /presto  /flink /alluxio  documents index  to the root of file ReadMe.md   ，it helps us to find the relevant integration documents

**Comment by qiuchenjian** [comment]
Please add describe to the title  like other PRS

File: README.md [hunk 1]
```diff
@@ -67,6 +67,11 @@ CarbonData is built using Apache Maven, to [build CarbonData](https://github.com
 * [Carbon as Spark's Datasource](https://github.com/apache/carbondata/blob/master/docs/carbon-as-spark-datasource-guide.md) 
 * [FAQs](https://github.com/apache/carbondata/blob/master/docs/faq.md) 
 
+##  Integrations
+* [Hive](https://github.com/apache/carbondata/blob/master/docs/hive-guide.md)
+* [Presto](https://github.com/apache/carbondata/tree/master/integration/presto)
+* [Flink]( )
```
Comment (xubo245) [inline README.md]: It should be add after there are related flink documentation

Comment (BeyondYourself) [reply README.md]: ok ,I get it

**Pull request closed without merging**